extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/MultipartTransformer.java

```diff
@@ -79,11 +82,45 @@ public void visitEnd() {
             // ctx param
             injectMethod.visitIntInsn(Opcodes.ALOAD, 1);
 
-            // call the populator
-            injectMethod.visitMethodInsn(Opcodes.INVOKESTATIC, populatorName.replace('.', '/'),
-                    DotNames.POPULATE_METHOD_NAME,
-                    String.format("(%s%s)V", thisDescriptor, INJECTION_CONTEXT_DESCRIPTOR), false);
-
```

Comment:
I debated whether to keep this around for launch modes that are not test, but I think there isn't too much benefit in doing so...